modules/javascript-udfs/pages/javascript-functions-with-couchbase.adoc

```diff
@@ -55,11 +70,107 @@ Arrow functions local to individual JavaScript functions are supported.
 
 Logging using the `console.log(..)` function is not supported.
 
-In the rest of this section, you're going to look at the concepts behind JavaScript User-Defined Functions:
+== Restricted Features
 
-* xref:calling-javascript-from-n1ql.adoc[]
-* xref:calling-n1ql-from-javascript.adoc[]
-* xref:handling-errors-javascript-udf.adoc[]
+[.status]#Couchbase Server 7.6.2#
+
+The following features are restricted in JavaScript functions for Query in Couchbase Server 7.6.2 and later.
+
+=== Code Injection
+
+JavaScript constructs that may allow for code injection or speculative execution attacks have been removed:
```

Comment:
The part about "speculative execution attacks" should be under the "Date Granularity" header and not the "Code Injection" header.

Reply:
Done